extensions/keycloak-authorization/runtime/src/main/java/io/quarkus/keycloak/pep/KeycloakPolicyEnforcerConfig.java

```diff
@@ -0,0 +1,192 @@
+package io.quarkus.keycloak.pep;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.keycloak.representations.adapters.config.PolicyEnforcerConfig;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.quarkus.runtime.annotations.DefaultConverter;
+
+@ConfigRoot(name = "keycloak", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public class KeycloakPolicyEnforcerConfig {
+
+    /**
+     * Adapters will make separate HTTP invocations to the Keycloak server to turn an access code into an access token.
+     * This config option defines how many connections to the Keycloak server should be pooled
+     */
+    @ConfigItem(defaultValue = "20")
+    int connectionPoolSize;
+
+    /**
+     * Policy enforcement configuration when using Keycloak Authorization Services
+     */
+    @ConfigItem
+    KeycloakConfigPolicyEnforcer policyEnforcer;
+
+    @ConfigGroup
+    public static class KeycloakConfigPolicyEnforcer {
+
+        /**
+         * Specifies how policies are enforced.
+         */
+        @ConfigItem
+        boolean enable;
+
+        /**
+         * Specifies how policies are enforced.
+         */
+        @ConfigItem(defaultValue = "ENFORCING")
+        String enforcementMode;
+
+        /**
+         * Specifies the paths to protect.
+         */
+        @ConfigItem
+        Map<String, PathConfig> paths;
```

Comment:
Is there any reason that this can't use the existing path stuff from our current authorization layer? The original idea was that the policies would be pluggable, and there would be one place to map paths to policies.

Reply:
The key point here is that we are not just plugging in a new policy, but a completely different solution to authorization.

I tried to do it using what Quarkus provides, but that would mean bringing too many things that are really specific to this feature in Keycloak.